api/employee/data-builder.js

```diff
@@ -22,7 +22,7 @@ const createFullName = employee => {
       suffix = employee['name.suffix'] || employee.suffix,
       fullName = '';
 
-  fullName += first.trim().replace(/[.]/g, '') || '';
+  fullName += first ? first.trim().replace(/[.]/g, '') : '';
```

Comment:
A little more succinct 
```suggestion
  fullName += (first || '').trim().replace(/[.]/g, '');
```